runtime/src/main/scala/org/apache/pekko/grpc/internal/GrpcResponseHelpers.scala

```diff
@@ -111,6 +111,6 @@ object GrpcResponseHelpers {
         headers.`Message-Encoding`(writer.messageEncoding.name) ::
         GrpcEntityHelpers.trailers(trailer.status, trailer.metadata),
       entity = HttpEntity.empty(writer.contentType)
-    )
+    ).withAttributes(Map(AttributeKeys.trailer -> trailer))
```

Comment:
if the trailer headers are already written to the `headers`, are they still needed as `attribute`?

Reply:
I think its nicer on the akka http level to have a unified way of finding the trailer regardless of the success or failure of the request, wdyt

Reply:
AFAICT this change would cause the status/metadata headers to be added twice: once to the regular response, and then once again as a block of trailing headers.

While I can understand the attraction of having the status information in the 'same place', I'm not sure that weights up against the optimization of having a regular set of headers rather than a split between normal and trailing headers in this case.

Reply:
Wait is that because I manually add them to the headers myself and if a trailer attribute gets detected it will also get added somewhere further in the pipeline?

Reply:
Wouldn't it be preferable to then only add it to the attributes and rely on it getting added later on?

Reply:
Can you point me to the place where this translation from trailer attributes to headers happens?